.github/workflows/unity_tests.yml

```diff
@@ -1,28 +1,27 @@
-name: Test Package
+name: Test Unity Package Workflow
 
 on:
   pull_request:
-    branches:
-      - main
+    branches: [main]
 
 jobs:
   testAllModes:
-    name: Test in ${{ matrix.testMode }}
+    name: Test Job for Unity Package in ${{ matrix.testMode }} 🕹️
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         projectPath:
         - Packages/io.chainsafe.web3-unity
         publishPath:
-        - ${GITHUB_WORKSPACE}/src/ChainSafe.GamingSdk.EVM.Unity/bin/Release/netstandard2.1/publish
+        - ${GITHUB_WORKSPACE}/src/ChainSafe.Gaming.Unity/bin/Release/netstandard2.1/publish
         packageLibraryPath:
         - ${GITHUB_WORKSPACE}/Packages/io.chainsafe.web3-unity/Runtime/Libraries
         testMode:
-          - playmode
-          - editmode
+          - PlayMode
```

Comment:
revert these camel case changes for editmode and playmode, they're variables set by [GameCI](https://game.ci/docs/github/test-runner)